platforms/svelte.ts

```diff
@@ -14,12 +14,15 @@ import { serve as serveBase } from "../src";
  */
 export const serve = <TInitialPayload = unknown>(
   routeFunction: RouteFunction<TInitialPayload>,
-  options: Omit<WorkflowServeOptions<Response, TInitialPayload>, "onStepFinish"> & {
-    env: WorkflowServeOptions["env"];
+  options: PublicServeOptions<TInitialPayload> & {
+    env: PublicServeOptions["env"]; // make env required
   }
 ): { POST: RequestHandler } => {
   const handler: RequestHandler = async ({ request }) => {
-    const { handler: serveHandler } = serveBase<TInitialPayload>(routeFunction, options);
+    const { handler: serveHandler } = serveBase<TInitialPayload>(routeFunction, {
+      ...options,
+      useJSONContent: true,
```

Comment:
I understand why it is a problem for express, but I don't understand why it caused a problem in svelte, can you add it to the pr description just for reference?

Reply:
added more info to description.